ClosedXML/Excel/Tables/XLTable.cs

```diff
@@ -817,5 +817,39 @@ public DataTable AsNativeDataTable()
 
             return table;
         }
+
+        public IXLTable CopyTo(IXLWorksheet targetSheet)
+        {
+            return CopyTo((XLWorksheet) targetSheet);
+        }
+
+        internal IXLTable CopyTo(XLWorksheet targetSheet)
+        {
+            String tableName = Name;
```

Comment:
Should copying a table not also involve copying the contents of the cells in the table, i.e. the range should be copied too?

Reply:
As long as we use this method in the scope of worksheet copying only it is not needed as cells are copied separately. But if we would like to use this method separately, we'd have to perform cells copying as well. Although I do not see any use cases that would require it.

Reply:
Well, you included it in `IXLTable.cs`, same with the other `CopyTo` methods. So people are going to use it. And I think it's a good idea to have this publicly exposed.